.github/workflows/update-protobuf.yml

```
@@ -0,0 +1,226 @@
+              echo "- Updated protobuf dependencies in buf.lock" >> $GITHUB_OUTPUT
+            fi
+            echo "- Regenerated Go code from latest protobuf definitions" >> $GITHUB_OUTPUT
+            echo "- Updated from a3m protobuf schema: buf.build/penwern/a3m" >> $GITHUB_OUTPUT
+            echo "" >> $GITHUB_OUTPUT
+            echo "_This PR was automatically created by the weekly protobuf update workflow._" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create Pull Request
+        if: steps.check-generated.outputs.has_generated_changes == 'true'
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)